lib/settings/_general.dart

```diff
@@ -254,6 +254,18 @@ class SettingsGeneralFragment extends StatelessWidget with AppBarMixin {
                 child: Text(L10n.of(context).large),
               ),
             ]),
+
+        ///TODO: translation etc.
+        const PrefSwitch(
+          pref: optionsMediaDefaultMute,
+          title: Text("Default muting"),
+          subtitle: Text("Mute media by default"),
+        ),
+        const PrefSwitch(
+          pref: optionMediaRememberMute,
```

Comment:
I don't think this need to be an option - it probably makes things a bit more confusing to the user. I think we should just automatically keep the user's current mute setting while a user is scrolling through a group feed.

Reply:
I really like this option - in fact there is an reddit app/client, called "infinity", where this is an option. And i got used to it pretty fast. And if i'm not wrong, the twitter App itself behaves this way. But i'm totally okay with it, removing it.